source/Node.js

```diff
@@ -441,23 +441,30 @@ function fixContainer ( container, root ) {
     return container;
 }
 
-function unparentedLi(node) {
-  return !node.parentNode ||
-         !['UL', 'OL'].includes(node.parentNode.tagName)
+function isParentedBy(node, parents=[]) {
+  if (!node.parentNode) {
+    return false
+  }
+
+  if (parents.includes(node.parentNode.tagName)) {
+    return true
+  }
+
+  return isParentedBy(node.parentNode, parents)
```

Comment:
Naming OCD guy here - the moment you start going up the tree, that's an `ancestor` rather than a parent. I know jquery has taken liberties with using parent, but it's really clear in commonplace English... `hasAncestor` is a decently clearer name for example.

Reply:
Jeez